iphone/cli/commands/_build.js

```diff
@@ -3448,6 +3454,11 @@ iOSBuilder.prototype.createXcodeProject = function createXcodeProject(next) {
 						return;
 					}
 
+					if (type === 'PBXShellScriptBuildPhase' && this.deployType !== 'production') {
+						this.logger.warn(`Only include ${type} in "${targetName}" for production, skipping now …`);
```

Comment:
This doesn't seem like a Titanium thing. Nevertheless, I think the wording could be less confusing.

Assuming the watch app project has a `PBXShellScriptBuildPhase`, then it will always display this warning for simulator/device builds. By saying "Only include" implies "Remove it" for non-production builds and that doesn't seem like a good recommendation.

Wouldn't it be better to just say "Excluding PBXShellScriptBuildPhase for non-production build"?